### DISCUSSION TITLE
Support for ogf in new Sonos

### DISCUSSION THREAD

**Discussion by philippe44** [philippe44/AirConnect#342] [General]
Does somebody know if new sonos devices support ogf? (flac inside an ogg container that would allow lossless playback together some metadata). My old PLAY:3 does not

Here is a link to try http://stream.radioparadise.com/flacm

**Comment by pwt** [comment]
Doesn't work properly on a modern, device (Sonos Port) running S2. Same as with an older device (Sonos Connect) running S1. Tries to start the stream, obtains metadata, but continually restarts with short bursts of noise.

(Just in case you don't know: in the specific case of Radio Paradise, there's a dedicated Sonos Music Service, including FLAC streams, metadata, and the ability to skip ahead.)

**Reply by philippe44** [reply]
Thanks - it's too bad, I was hoping that ogf could give metadata and lossless for Sonos. Some UPnP devices support ogf, but truly, if Sonos does not I'm not sure I'll bother adding it as a possible codec option.

I knew about the RP option for Sonos, thanks, but I was trying to find a generic improvement for AirConnect
